src/UhdmAst.cpp

```diff
@@ -298,6 +324,7 @@ AstBasicDTypeKwd get_kwd_for_type(int vpi_var_type) {
     case vpiStructTypespec:
     case vpiStructNet:
     case vpiStructVar:
+    case vpiPackedArrayTypespec:
```

Comment:
Is there a situation where we would want a basic type keyword for this? If not, shouldn't we return an error instead?

Reply:
I don't know if there can be such situation. I thought that this case is for complex data types. I thought that `vpiPackedArrayTypespec` should be treated similar to `vpiStructTypespec` in such situations.